superagi/agent/agent_iteration_step_handler.py

```diff
@@ -140,11 +148,14 @@ def _build_agent_prompt(self, iteration_workflow: IterationWorkflow, agent_confi
     def _build_tools(self, agent_config: dict, agent_execution_config: dict):
         agent_tools = [ThinkingTool()]
 
-        model_api_key = AgentConfiguration.get_model_api_key(self.session, self.agent_id, agent_config["model"])
+        config_data = AgentConfiguration.get_model_api_key(self.session, self.agent_id, agent_config["model"])
+        model_api_key = config_data['api_key']
         tool_builder = ToolBuilder(self.session, self.agent_id, self.agent_execution_id)
-        resource_summary = ResourceSummarizer(session=self.session,
-                                              agent_id=self.agent_id).fetch_or_create_agent_resource_summary(
-            default_summary=agent_config.get("resource_summary"))
+        print("%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%")
```

Comment:
remove prints

Reply:
ok sir surely will remove them before merging it to dev